framework/Files.py

```diff
@@ -523,6 +523,28 @@ class StaticXMLOutput(RAVENGenerated):
   """
     Specialized class for consistent RAVEN XML outputs.  See forms in comments above.
   """
+
+  def __getstate__(self):
+    """
+      Pickle dump method hook.
+      @ In, None
+      @ Out, statedict, dict, dict of objects needed to restore instance
+    """
+    statedict = RAVENGenerated.__getstate__(self)
```

Comment:
```stateDict```?

Reply:
This is following convention used elsewhere in RAVEN and python in general.  If we want to make an issue to change it in other files, I'm fine with that, but I'd rather be consistent and don't have time to check for it everywhere.

Reply:
Personally, I think we should respect RAVEN standards as much as we can, and I prefer to opening an issue.

Reply:
Okay, I spent the time.